src/client/pages/PasscodeEmailSent.tsx

```diff
@@ -112,6 +112,7 @@ export const PasscodeEmailSent = ({
 	const [recaptchaErrorMessage, setRecaptchaErrorMessage] = useState('');
 	const [recaptchaErrorContext, setRecaptchaErrorContext] =
 		useState<ReactNode>(null);
+	const formRef = createRef<HTMLFormElement>();
```

Comment:
Should we use `useRef` instead as we're using function components 
 as per 
https://react.dev/reference/react/createRef

<img width="916" alt="image" src="https://github.com/user-attachments/assets/0b77904d-f17c-4e29-9606-e9f0aeaa6f45" />

Reply:
Ah good spot! I'll try and refactor it!